packages/remark-parse/lib/tokenize/list.js

```diff
@@ -139,7 +141,13 @@ function list(eat, value, silent) {
 
   character = value.charAt(++index);
 
-  if (character !== C_SPACE && character !== C_TAB) {
+  delimiters = {};
+  delimiters[C_SPACE] = true;
+  delimiters[C_TAB] = true;
+  delimiters[C_NEWLINE] = commonmark;
+  delimiters[C_EMPTY] = commonmark;
```

Comment:
With delimiters empty being checked, how will

```markdown
1.2.3.
```

be parsed?

Reply:
It will just be text.  

If you look on line [142](https://github.com/remarkjs/remark/pull/328/files/f3c06ab664be596d89729c7cd50fbe12626d29af#diff-0b27a32611df78dd18ed7a0b170ec571R142), you can see we grab the next `character` to see if it is a valid delimiter.  This is the only character compared against the delimiters and the only time it would be empty is when there is no next character (the end of the file has been reached).

So in the case of `1.2.3.`, `2` would be the next character after matching the first list-item-looking-thing and that's not a valid delimiter, so we'd early return and it would not be parsed as a list.